File: deno/src/base/dictbiz_detail/dictbiz_detail.service.ts
import * as dict_detailDao from "./dictbiz_detail.dao.ts";

/**
 * 获取 codes 对应的业务字典
 */
export async function getDictbiz(
  codes: string[] = [ ],
) {
  return await dict_detailDao.getDictbiz(codes);
}

File: rust/src/src/base/dictbiz_detail/dictbiz_detail_service.rs
use color_eyre::eyre::Result;

use super::dictbiz_detail_model::GetDictbiz;
use super::dictbiz_detail_dao;

pub async fn get_dictbiz(
  codes: Vec<String>,
) -> Result<Vec<Vec<GetDictbiz>>> {
  
  let data = dictbiz_detail_dao::get_dictbiz(
    codes.as_slice(),
  ).await?;
  
  Ok(data)
}
